.github/workflows/build.yml

```diff
@@ -28,113 +156,27 @@ jobs:
         with:
           url: ${{ env.CPP_SDK_URL }}
           tag: ${{ env.CPP_SDK_TAG }}
-          asset_name: "cppsdk-${{ env.CPP_SDK_TAG }}-${{ env.BUILD_PLATFORM }}-universal.zip"
-      - name: Remove symlinks from cpp sdk libs
+          asset_name: "cppsdk-${{ env.CPP_SDK_TAG }}-${{ env.BUILD_PLATFORM }}.zip"
+      - name: Unpack AAR
         env:
-          LIB_PATH_UNIVERSAL: "${{ github.workspace }}/${{ env.CPP_SDK_DESTINATION }}/sdk-release/lib"
+          AAR_PATH_BASE: "${{ github.workspace }}/${{ env.CPP_SDK_DESTINATION }}/comms-sdk-android-cppsdk-"
+          AAR_PATH: "${{ env.AAR_PATH_BASE }}${{ env.CPP_SDK_TAG }}.aar"
+          AAR_NO_PREFAB_PATH: "${{ env.AAR_PATH_BASE }}no-prefab-${{ env.CPP_SDK_TAG }}.aar"
```

Comment:
Should be noprefab instead of no-prefab.

Reply:
yup, little typo here